aten/src/TH/generic/THTensorLapack.cpp

```diff
@@ -191,88 +191,6 @@ void THTensor_(gels)(THTensor *rb_, THTensor *ra_, THTensor *b, THTensor *a)
   if (free_b) c10::raw::intrusive_ptr::decref(b);
 }
 
-void THTensor_(geev)(THTensor *re_, THTensor *rv_, THTensor *a_, bool eigenvectors)
-{
-  char jobvr = eigenvectors ? 'V' : 'N';
-  int n, lda, lwork, info, ldvr;
-  THTensor *work=nullptr, *wi, *wr, *a;
-  scalar_t wkopt;
-  scalar_t *rv_data;
-  int64_t i;
-
-  THTensor *re__ = NULL;
-  THTensor *rv__ = NULL;
-
-  THArgCheck(a_->dim() == 2, 1, "A should be 2 dimensional");
-  THArgCheck(a_->size(0) == a_->size(1), 1,"A should be square");
-  THArgCheck(THTensor_(isFinite)(a_), 1, "A should not contain infs or NaNs");
-
-  /* we want to definitely clone a_ for geev*/
-  a = THTensor_(cloneColumnMajor)(NULL, a_);
-
-  n = a->size(0);
-  lda = n;
-
-  wi = THTensor_(newWithSize1d)(n);
-  wr = THTensor_(newWithSize1d)(n);
-
-  rv_data = NULL;
-  ldvr = 1;
-  if (jobvr == 'V')
-  {
-    THTensor_(resize2d)(rv_,n,n);
-    /* guard against someone passing a correct size, but wrong stride */
-    rv__ = THTensor_(newTransposedContiguous)(rv_);
-    rv_data = rv__->data<scalar_t>();
-    ldvr = n;
-  }
-  THTensor_(resize2d)(re_,n,2);
-  re__ = THTensor_(newContiguous)(re_);
-
-  if (n > 0) {  // lapack doesn't work with size 0
-    /* get optimal workspace size */
-    THLapack_(geev)('N', jobvr, n, a->data<scalar_t>(), lda, wr->data<scalar_t>(), wi->data<scalar_t>(),
-        NULL, 1, rv_data, ldvr, &wkopt, -1, &info);
-
-    lwork = (int)wkopt;
-    work = THTensor_(newWithSize1d)(lwork);
-
-    THLapack_(geev)('N', jobvr, n, a->data<scalar_t>(), lda, wr->data<scalar_t>(), wi->data<scalar_t>(),
```

Comment:
looks like you no longer need THLapack_(geev), can you delete it too?

Reply:
Thanks, I forgot there were multiple geev functions in different places.